### DISCUSSION TITLE
Can set timezone with localtime for rook?

### DISCUSSION THREAD

**Discussion by zhucan** [rook/rook#11142] [Q&A]
The timezone of the container is usa, but the localtime is Asia/Shanghai. The ceph pg scrub will be effected by the timezone.

**Comment by travisn** [comment]
Rook doesn't set timezone or time, it should just pick it up from the container runtime. What is the timezone and localtime of your host?

**Reply by zhucan** [reply]
Asia/Shanghai @travisn

**Reply by travisn** [reply]
Do you see other pods besides rook and ceph with the usa time zone? Not sure why it wouldn't pick up the host time zone.

**Comment by zhucan** [comment]
for example: all of csi pods use usa time zone but we can mount the localtime to the pods

**Reply by zhangdeshuai1999** [reply]
Did you solve the time zone problem? Same for me

**Reply by zhucan** [reply]
need to modify the  rook code by yourself. @zhangdeshuai1999

**Reply by zhangdeshuai1999** [reply]
Image reconstruction?

**Reply by zhucan** [reply]
yes, You need to maintain this part of the code yourself

**Reply by travisn** [reply]
@zhucan What did you modify?
